actors/market/src/lib.rs

```diff
@@ -554,7 +562,7 @@ impl Actor {
             let mut deal_states: Vec<(DealID, DealState)> = vec![];
             let mut batch_gen = BatchReturnGen::new(params.sectors.len());
             let mut activations: Vec<SectorDealActivation> = vec![];
-            let mut activated_deals = BTreeSet::<DealID>::new();
+            let mut activated_deals: HashSet<DealID> = HashSet::new();
```

Comment:
```suggestion
            let mut activated_deals = HashSet::<DealID>::new();
```

Reply:
or not .. I see some inconsistency below, like in actors/miner/tests/util.rs where there's a lot of explicit variable typing so I guess we don't have a rule in here (yet)